### PR TITLE
creating the 404 page widget area

### PR DESCRIPTION
**Pull request by viscosho**
creating the 404 page widget area

### CODE REVIEW

File: 404.php [hunk 1]
```diff
@@ -23,4 +23,14 @@
...
 
-<?php get_footer();
+	<div id="secondary" class="col-sm-12 sidebar-container" role="complementary">
+		<div class="widget-area">
+			<?php dynamic_sidebar( 'sidebar-404' ); ?>
+		</div><!-- .widget-area -->
+	</div><!-- #secondary -->
+
+<script type="text/javascript">
+	jQuery('.search-field').attr('placeholder', 'lsdev.biz')
```
Comment (nandotess) [inline 404.php]: @viscosho please never use inline JavaScript in our core software. On child theme is also not recommended, but I understand sometimes it's necessary. But here we need follow the best practices and in that case, we need to move this code to the LSX core JavaScript file. In this JavaScript, please test if you are on the 404 page before running this line of code.

File: includes/404-widget.php [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+/**
+ * LSX functions and definitions - Config.
+ *
+ * @package    lsx
+ * @subpackage 404-widget
+ */
+
+function lsx_widget_area_404_init() {
+	register_sidebar( array(
+			'name'          => esc_html__( '404 page', 'lsx' ),
```
Comment (nandotess) [inline includes/404-widget.php]: @viscosho please run the gulp tasks related to the translation files. As we can see here, you have a new test that is marked as translatable, so we need to make sure the LSX translate files are updated.

File: includes/404-widget.php [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * LSX functions and definitions - Config.
+ *
+ * @package    lsx
+ * @subpackage 404-widget
+ */
+
+function lsx_widget_area_404_init() {
```
Comment (nandotess) [inline includes/404-widget.php]: @viscosho please check if the function exists before to create it. Check the other include files to see how we do it in LSX core theme.

File: includes/404-widget.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+			'before_widget' => '<aside id="%1$s" class="widget %2$s">',
+			'after_widget'  => '</aside>',
+			'before_title'  => '<h3 class="widget-title">',
+			'after_title'   => '</h3>',
+		) );
+	}
+
+add_action( 'widgets_init', 'lsx_widget_area_404_init' );
+
+function my_search_placeholder() {
```
Comment (nandotess) [inline includes/404-widget.php]: @viscosho please check if the function exists before to create it. Check the other include files to see how we do it in LSX core theme.

**Comment by krugazul** [comment]
Pull Request Merged

**Pull request merged**